docs/API/core/storage.md

```diff
@@ -81,7 +81,7 @@ Fliplet.App.Storage.remove('key').then(function () {
 
 ## Namespaced storage
 
-You can also create a private namespaced storage which is nor shared with the `Fliplet.App.Storage` neither with the global `Fliplet.Storage`:
+You can also create a private namespaced storage which is not shared with the `Fliplet.App.Storage` neither with the global `Fliplet.Storage`:
```

Comment:
`nor` is actually OK to use in this sentence but the order (nor, neither) should be swapped to neither, nor

Reply:
Updated